### DISCUSSION TITLE
How can I pay pro version with EUR instead?

### DISCUSSION THREAD

**Discussion by guoliang** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#865] [Q&A] [answered]
Hi, I'm interested to purchase a license for the pro version however the option I get to pay in isn't in the currency my card is in. I'm not interested in paying some exchange fees so I'm wondering how I can change so I can pay in the currency my card is in?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @guoliang,

The currency during the payment process is based on the country you access the internet from. However, if you get the wrong country, you can choose the correct one during the checkout process.

Cheer,
Matthias
